ui/main/src/app/services/processes.service.ts

```diff
@@ -109,6 +110,12 @@ export class ProcessesService {
                 );
     }
 
+    private loadAllProcceseData() {
+        this.processes.forEach(process => {
+            this.queryProcess(process.id, process.version).subscribe();
```

Comment:
I think , you do not need to make queries to the back as the information is already present in the process object . 
So just doing : 
```
 this.processes.forEach(process => {
this.processCache.set(`${process.id}.${process.version}` ,process); 
...
```
would do the job ? 

In this case , the method shall be rename `loadAllProcessesInCache()"

Reply:
Done